### PR TITLE
Script to enable mx completion on zsh

### PR DESCRIPTION
**Pull request by zakkak**
`source mx.zsh` enables zsh completion for mx

### CODE REVIEW

**Comment by zakkak** [comment]
Sorry for the inconvenience, a signed form has been sent to oracle-ca_us [at] oracle [dot] com

File: bash_completion/mx.zsh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# source this file to have zsh completion for mx
+
+autoload -U bashcompinit
```
Comment (dougxc) [inline bash_completion/mx.zsh]: is `bashcompinit` a script somewhere?

Comment (mukel) [reply bash_completion/mx.zsh]: Just a `zsh` function to enable compatibility with bash completions.

I'd just put it in my ~/.zshrc like this
```bash
autoload -U bashcompinit
source $MX_HOME/bash_completions/mx
```

**Comment by mukel** [comment]
The bash completions provided are very basic.

You can get better completions for mx options in `zsh` by just executing `compdef _gnu_generic mx`.
Sub-command completions are just a few lines of parsing.

If you use `zsh` you should definitely try [:honey_pot: sweet (zsh) completions for mx](https://github.com/mukel/mx-honey).

File: bash_completion/mx.zsh (hunk 1 above)
Comment (dougxc) [reply bash_completion/mx.zsh]: Yes, this looks like something better kept out of repo and under direct control of the user.

Comment (zakkak) [reply bash_completion/mx.zsh]: OK, then maybe you can close this issue and put a reference to https://github.com/mukel/mx-honey in the readme file

Comment (dougxc) [reply bash_completion/mx.zsh]: @mukel can you please take care of that.

**Pull request closed without merging**